### PR TITLE
Opportunity to change admin accounts 

### PR DESCRIPTION
**Pull request by sunmoon11100**

### CODE REVIEW

File: sources/swap/config.move [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    const MIN_FEE: u64 = 1;
+
+    /// Maximum value of fee.
+    const MAX_FEE: u64 = 35;
+
+    /// Minimum value of dao fee.
+    const MIN_DAO_FEE: u64 = 0;
+
+    /// Maximum value of dao fee.
+    const MAX_DAO_FEE: u64 = 100;
```
Comment (mkurnikov) [inline sources/swap/config.move:31]: Now you have duplicated set of constants? The same ones are present in `liquidity_pool.move`

Comment (mkurnikov) [reply sources/swap/config.move:31]: Add `is_valid_fee` and `is_valid_dao_fee` functions.

File: sources/swap/config.move [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    /// Initializes admin contracts when initializing the liquidity pool.
+    public(friend) fun initialize(liquidswap_admin: &signer) {
+        move_to(liquidswap_admin, PoolConfig {
+            dao_admin_address: @dao_admin,
+            emergency_admin_address: @emergency_admin,
+            fee_admin_address: @fee_admin,
+            default_uncorrelated_fee: 30,
+            default_stable_fee: 4,
+            default_dao_fee: 33,
```
Comment (mkurnikov) [inline sources/swap/config.move:50]: Add comments that it's 0.3%, 0.04% and 33%

**Review by mkurnikov — CHANGES_REQUESTED** [review]

**Review by borispovod — APPROVED** [review]
Well, let's merge all together and after i will share opinion.
Currently i see minor issues in this branch, probably fixed in `dynamic-fees`.

**Pull request merged**